JL.Windows/GUI/PopupWindow.xaml.cs

```diff
@@ -695,6 +699,29 @@ private StackPanel PrepareResultStackPanel(LookupResult result, int index, int r
             audioButton.Click += AudioButton_Click;
 
             _ = top.Children.Add(audioButton);
+
+            if (!configManager.MineToFileInsteadOfAnki)
+            {
+                Button duplicate = new()
```

Comment:
```suggestion
                TextBlock duplicateIconTextBlock = new()
```

This can be a `TextBlock` since we don't expect it to do anything when it's clicked.